app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
+end
```

Comment:
- [ ]  Great  job to create User table, but seemed like you missed something based on provided ERD Diagram.  You've shown how user will be able to create the posts, but you forget that the same user might create comments and Likes . Please create the association in which a user can create comments and likes as well.